depclean-maven-plugin/src/test/java/se/kth/depclean/DepCleanMojoIT.java

```diff
@@ -19,11 +18,31 @@
 public class DepCleanMojoIT {
 
   @MavenTest
-  @MavenGoal("package")
   @DisplayName("Test that DepClean runs in an empty Maven project")
   void empty_project(MavenExecutionResult result) {
     assertThat(result).isFailure(); // should pass
   }
 
+  @MavenTest
+  @DisplayName("Test that DepClean runs in a Maven project with processors")
+  void processor_used(MavenExecutionResult result) {
+    assertThat(result).isSuccessful().out()
+            .plain().contains(
+            "-------------------------------------------------------",
+            " D E P C L E A N   A N A L Y S I S   R E S U L T S",
+            "-------------------------------------------------------",
+            "USED DIRECT DEPENDENCIES [1]: ",
+            "	org.mapstruct:mapstruct-processor:1.4.2.Final:provided (1 MB)",
+            "USED INHERITED DEPENDENCIES [0]: ",
+            "USED TRANSITIVE DEPENDENCIES [1]: ",
+            "	com.fasterxml.jackson.core:jackson-core:2.12.2:compile (356 KB)",
+            "POTENTIALLY UNUSED DIRECT DEPENDENCIES [1]: ",
+            "	com.fasterxml.jackson.core:jackson-databind:2.12.2:compile (1 MB)",
+            "POTENTIALLY UNUSED INHERITED DEPENDENCIES [0]: ",
+            "POTENTIALLY UNUSED TRANSITIVE DEPENDENCIES [1]: ",
+            "	com.fasterxml.jackson.core:jackson-annotations:2.12.2:compile (73 KB)"
+    );
+  }
+
 }
```

Comment:
Nice that you included a proper IT!

Reply:
I had a very hard time debugging the plugin though, maybe I'm doing something wrong... I was unable to catch a breakpoint in the core project.

With more ITs, this big assertion can be refined, though.